bundles/org.openhab.binding.electroluxappliance/README.md

```diff
@@ -0,0 +1,129 @@
+| door-lock                    | Contact               | This channel reports the status of the door lock.                              |
+| time-to-start                | Number:Time           | This channel reports the remaining time for a delayed start washing program.   |
+| time-to-end                  | Number:Time           | This channel reports the remaining time to the end for a washing program.      |
+| cycle-phase                  | String                | This channel reports the washing cycle phase.                                  |
+| analog-temperature           | String                | This channel reports the washing temperature.                                  |
+| steam-value                  | String                | This channel reports the washing steam value.                                  |
+| programs-order               | String                | This channel reports the washing program.                                      |
+| analog-spin-speed            | String                | This channel reports the washing spin speed.                                   |
+| appliance-state              | String                | This channel reports the appliance state.                                      |
+| appliance-mode               | String                | This channel reports the appliance mode.                                       |
+| appliance-total-working-time | Number:Time           | This channel reports the total working time for the washing machine.           |
+| appliance-ui-sw-version      | String                | This channel reports the appliance UI SW version.                              |
+| optisense-result             | String                | This channel reports the optisense result.                                     |
+| detergent-extradosage        | String                | This channel reports the detergent extra dosage.                               |
+| softener-extradosage         | String                | This channel reports the softener extra dosage.                                |
+| water-usage                  | Number:Volume         | This channel reports the water usage in litres.                                |
+| total-wash-cycles-count      | Number                | This channel reports the total number of washing cycles.                       |
+| status                       | String                | This channel is used to fetch latest status from the API.                      |
+
+
+## Full Example
+
+### `demo.things` Example
+
+```java
+// Bridge configuration
+Bridge electroluxappliances:api:myAPI "Electrolux Group API" [apiKey="12345678", accessToken="12345678", refreshToken="12345678", refresh="300"] {
+
+     Thing air-purifier myair-purifier  "Electrolux Pure A9"    [ serialNummber="123456789" ]
+     
+}
+```
+
+##  `demo.items` Example
+
+```java
```

Comment:
Update this example with: electroluxappliances => electroluxappliance

Reply:
Fixed!